subsys/net/lib/lwm2m/lwm2m_obj_portfolio.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME net_lwm2m_obj_portfolio
+#define LOG_LEVEL CONFIG_LWM2M_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <stdint.h>
+#include <init.h>
+
+#include "lwm2m_object.h"
+#include "lwm2m_engine.h"
+#include "lwm2m_resource_ids.h"
+
+#define PORTFOLIO_VERSION_MAJOR 1
+#define PORTFOLIO_VERSION_MINOR 0
+
+#define PORTFOLIO_IDENTITY_ID			0
+#define PORTFOLIO_GET_AUTH_DATA_ID		1
+#define PORTFOLIO_AUTH_DATA_ID			2
+#define PORTFOLIO_AUTH_STATUS_ID		3
+
+#define PORTFOLIO_MAX_ID 4
+
+/* Identity max based on LwM2M v1.1 conformance test requirements */
+#define PORTFOLIO_IDENTITY_MAX 4
```

Comment:
These seem to be usually Kconfig-options, don't know if it matters here as it is for conformance testing.